openassessment/templates/openassessmentblock/oa_grade_available_responses.html

```diff
@@ -0,0 +1,32 @@
+{% load i18n %}
+{% spaceless %}
+<div class="wrapper wrapper--xblock wrapper--openassessment theme--basic">
+    <div class="openassessment problem">
+        <div class="wrapper--grid">
+            {% if title %}
+                <h3 class="openassessment__title problem__header">{% trans title %}</h3>
```

Comment:
Please make sure that the header levels are correct (I see h3 here and an h2 below), taking into account headers that are already present on the page.

@staubina can you advise what header level this should start with, given the work you are doing to combine the top-level h1 and h2?